test/rulesets/Base/indexing.jl

```diff
@@ -89,7 +89,7 @@
         
         test_rrule(getindex, Symmetric(rand(3, 3)), 2, 2)
         sgrad = rrule(getindex, Symmetric(rand(3, 3)), 2, 3)[2](1.0)[2]
-        @test unthunk(sgrad) ≈ [0 0 0; 0 0 1/2; 0 1/2 0]
+        @test unthunk(sgrad) ≈ [0 0 0; 0 0 1/2; 0 1/2 0]  # We are actually getting this wrong now
```

Comment:
It combines them to just be a `OneElement`
I think this might be a decent argument against `OneElement` at least in cases other than known dense arrays?

Reply:
Why does this fail? I'd expect ProjectTo would make an Array here.

If we do this, quite a few `(::ProjectTo)(::OneElement)` could specialise.

Reply:
We don't go through the `ProjectTo` right now Maybe we should.